SECURITY.md

```diff
@@ -1,3 +1,90 @@
+# Security Issue Response Process
 
-If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/) or directly via email to aws-security@amazon.com. Please do **not** create a public GitHub issue.
\ No newline at end of file
+## Introduction
+
+Security issues happen as part of the normal lifecycle of software development of OpenSearch. This document describes the process for reporting these issues, responding to these issues, and ensuring we treat them not only with the appropriate priority but also with respect for the discoverers, software providers and their users, and the OpenSearch community in general.
+
+If you discover a potential security issue in this project we ask that you notify the OpenSearch Security Team via email to opensearch-security@amazon.com. Please do **not** create a public GitHub issue.
+
+*Giving credit where credit is due, this policy is heavily influenced by the [Xen Project’s security response process](https://xenproject.org/developers/security-policy/), that was put to the test during the [embargo period for XSA-108 back in 2014](https://xenproject.org/2014/10/22/xen-project-security-policy-improvements-get-involved/) and improved its clarity around managing the pre-disclosure list and the deployment of fixes during embargo. We are standing on the shoulders of these battle-tested giants.*
+
+## The Security Team
+
+The OpenSearch Security Team is a subset of the project’s maintainers responsible for looking after the project’s security, including the security issue response process outlined below. Maintainers can be added to the Security Team by submitting a PR updating this document and adding their name to the list below (the process for becoming a maintaner can be found [here](https://github.com/opensearch-project/.github/blob/main/MAINTAINERS.md#becoming-a-maintainer)).
+
+The OpenSearch Security Team will address reported issues on a best effort basis, prioritizing them based on several factors, including severity.
+
+### Current Members
+
+| Security Team member     | GitHub Alias                                  | Affiliation |
+| ------------------------ | --------------------------------------------- | ----------- |
+| Dave Lago                | [davidlago](https://github.com/davidlago)     | Amazon      |
+| Charlotte Henkle         | [CEHENKLE](https://github.com/CEHENKLE)       | Amazon      |
+| Daniel (dB.) Doubrovkine | [dblock](https://github.com/dblock)           | Amazon      |
+| Anirudha (Ani) Jadhav    | [anirudha](https://github.com/anirudha)       | Amazon      |
+| Sean Neumann             | [seanneumann](https://github.com/seanneumann) | Amazon      |
+| Anan Zhuang              | [ananzh](https://github.com/ananzh)           | Amazon      |
+| Eli Fisher               | [elfisher](https://github.com/elfisher)       | Amazon      |
+
+## Process
```

Comment:
since you mention the [Xen Project’s security response process](https://xenproject.org/developers/security-policy/), i think you should consider including their step 3b:

> If we think other software systems (for example, competing hypervisor systems) are likely to be affected by the same vulnerability, we will try to make those other projects aware of the problem and include them in the advisory preparation process.

obviously in the beginning there's a big chance that problems also affect ES; though over time that number will probably go down as the code bases diverge.

Reply:
I'm assuming this clause is to capture non-list members who could be affected. We are establishing some level of vetting for projects to join the list, but potentially short-circuiting that process if a project is potentially affected but not already on the list. I have some concerns around this short circuit especially as it pertains to the other project's security problem handling processes (see pre-disclosure application process section).

As this paragraph describes a "best effort" approach (_if we think, likely to be affected, we will try_...) I would rather leave out for now. wdyt?